### DISCUSSION TITLE
ox-hugo exporting freezes

### DISCUSSION THREAD

**Discussion by gnohz** [kaushalmodi/ox-hugo#651] [Q&A]
When I try to export the org, there is message "[ox-hugo] exporting ..." and the buffer freezes for couple of seconds that I cannot do anything but wait. This happens when I use Doom Emacs, but when I am in Spacemacs, the export is instant. Any one has similar problems?

**Comment by kaushalmodi** [comment]
This looks similar to https://github.com/kaushalmodi/ox-hugo/discussions/551.

Is it possible that Doom emacs is using the Org mode from main branch (development version), while Spacemacs is using stable version of Org mode?

**Reply by gnohz** [reply]
Not sure, but read the discussions you pointed to. 
`(setq org-element-use-cache nil)` seems do the trick.
